projects/aca-content/src/lib/ui/variables/variables.scss

```diff
@@ -74,7 +78,11 @@ $defaults: (
   --theme-action-button-text-color: $action-button-text-color,
   --theme-page-layout-header-background-color: $page-layout-header-background-color,
   --theme-search-chip-icon-color: $search-chip-icon-color,
-  --theme-disabled-chip-background-color: $disabled-chip-background-color
+  --theme-disabled-chip-background-color: $disabled-chip-background-color,
+  --adf-metadata-property-panel-border-color: $adf-metadata-property-panel-border-color,
+  --adf-metadata-buttons-background-color: $adf-metadata-buttons-background-color,
+  --adf-snackbar-message-background-color: $adf-snackbar-message-background-color,
+  --adf-snackbar-message-color: $adf-snackbar-message-color
```

Comment:
this does not seem right... the snackbar colors should already be defined elsewhere, and your PR should be focusing on aspects, not changing other component colors

Reply:
I have reverted snackbar colors